e2e/_suites/fleet/stand-alone.go

```diff
@@ -69,6 +70,12 @@ func (sats *StandAloneTestSuite) aStandaloneAgentIsDeployed(image string) error
 		profileEnv["elasticAgentDockerImageSuffix"] = "-" + image
 	}
 
+	profileEnv["elasticAgentDockerNamespace"] = "beats"
+	useCISnapshots, _ := shell.GetEnvBool("ELASTIC_AGENT_USE_CI_SNAPSHOTS")
```

Comment:
Should documentation be added to indicate which environment variables can be set and how they can be used?

Reply:
There is an example in the README, although I agree and see it convenient to add a dedicated section for setting the environment. Let's do this in a follow-up PR

Thanks for the suggestion!